src/main/kotlin/no/nav/familie/ef/sak/api/beregning/Beregning.kt

```diff
@@ -16,550 +16,549 @@ data class Beregningsgrunnlag(val inntekt: BigDecimal,
                               val grunnbeløp: BigDecimal)
 
 data class Grunnbeløp(val fraOgMedDato: LocalDate,
-                      val tilDato: LocalDate,
+                      val tilOgMedDato: LocalDate,
```

Comment:
Burde man hatt med `val tilDato: LocalDate = tilOgMedDato` også, sånn att det ikke brekker for frontend før `tilDato` er slettet?

Reply:
Tror ikke grunnbeløp går ut til noen frontend, nei.